webapp/imports/ui/Home.jsx

```diff
@@ -94,8 +103,9 @@ class Home extends Component {
 
 export default createContainer(() => {
   Session.get("showGamesLimit") ? '' : Session.set("showGamesLimit", 4)
+  let fetchGame = Meteor.subscribe('games')
   return {
-    subscription: Meteor.subscribe('games').ready(),
+    loading: fetchGame.ready(),
```

Comment:
Should be called `loaded` if it's linked to `fetchGame.ready()`. Otherwise it should be `!fetchGame.ready()`